### ISSUE TITLE
`\<` and `\>` are incorrectly HTML-escaped as \&lt; and \&gt;

### ISSUE THREAD

**Issue by agorf** [notable/notable#249]
### Current behavior

In a note, write **both**

```
`\<`
```

and

```
`\>`
```

They are rendered as `\&lt;` and `\&gt;`

### Expected behavior

They should be rendered as `\<` and `\>`

### Environment

- **OS**: Debian GNU/Linux
- **Notable version**: 1.1.0

**Comment by danielhoherd** [comment]
More test cases:

Input:
```
Flew to Vietnam. SFO -> LAX -> NRT -> bus to HND -> SGN.
```

Output:
```
Flew to Vietnam. SFO -\gt ; L A X ->;LAX−gt; NRT -\gt ; b u s \to H N D ->;bus→HND−gt; SGN.
```

**Comment by fabiospampinato** [comment]
Overall this sounds like an issue with [showdown](https://github.com/showdownjs/showdown) maybe 🤔 /cc @tivie what do you think? should I open a separate bug report in showdown?

> Flew to Vietnam. SFO -> LAX -> NRT -> bus to HND -> SGN.

It looks like this is fixed in the master branch.

**Comment by tivie** [comment]
humm... Inside code spans or blocks, Showdown always escapes `<`, `>` and `&`. I've just tested this and is working as intended on Showdown's side.

**Comment by danielhoherd** [comment]
The example I gave was not inside a code block. It was part of a simple markdown list, such as:

```
# Some travel timeline
- Took BART to SFO
- Flew to Vietnam. SFO -> LAX -> NRT -> bus to HND -> SGN.
- Took a bus to An Giang
```

**Comment by tivie** [comment]
@danielhoherd I was referring to @mikaelmello example. 

Regardless, with your example,  Showdown's behavior is:

**input**:
```
Flew to Vietnam. SFO -> LAX -> NRT -> bus to HND -> SGN.
```

**output**:
```
<p>Flew to Vietnam. SFO -&gt; LAX -&gt; NRT -&gt; bus to HND -&gt; SGN.</p>
```

which is correct

**Comment by danielhoherd** [comment]
@tivie Github renders it just fine. https://gist.github.com/danielhoherd/18c2775ceb2174dac25ed4676e4c45fe

Notable does not.
<img width="1295" alt="Screen Shot 2019-03-19 at 9 57 43 AM" src="https://user-images.githubusercontent.com/1323808/54625948-76492680-4a2d-11e9-9d20-81d2e02ecd91.png">

**Comment by fabiospampinato** [comment]
Fixed in v1.6.
